transforms/language/pii_redactor/README.md

```diff
@@ -0,0 +1,10 @@
+# PII Redactor Transform
+
+* [python](python/README.md) - provides the base python-based transformation 
+implementation.
+* [ray](ray/README.md) - enables the running of the base python transformation
+in a Ray runtime
+* [spark](spark/README.md) - enables the running of a spark-based transformation
+in a Spark runtime. 
```

Comment:
```suggestion
<!-- Consider commenting out since we do not have a spark transform for this.
* [spark](spark/README.md) - enables the running of a spark-based transformation
in a Spark runtime. 
```